CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # bedrock-authn-token ChangeLog
 
+## 4.0.0 - 2021-04-33
+### Added
+- **BREAKING**: Added `combinedHash` function to assist with concatenating `clientId` and `hash` for `prefixedHash`.
+- **BREAKING**: Added `verifyHash` functions to assist with comparing `prefixedHash`, and `sha256`.
```

Comment:
These aren't public APIs so they shouldn't be mentioned in the changelog -- it will be confusing.